infoCourse.json

```diff
@@ -2,16 +2,124 @@
     "uuid": "fe472cd2-4759-4c0d-9b19-7383ca9b2817",
     "name": "TPL 101",
     "title": "Template Course",
-    "assessmentSets": [
-      {"abbreviation": "G", "name": "Gallery", "heading": "Gallery of questions using different PrairieLearn elements", "color": "orange1", "description": "Gallery of PL questions"}
-    ],
     "options": {
         "useNewQuestionRenderer": true
     },
+    "assessmentSets": [
+        {
+            "abbreviation": "G",
+            "name": "Gallery",
+            "heading": "Gallery of questions using different PrairieLearn elements",
+            "color": "orange1",
+            "description": "Gallery of questions"
+        },
```

Comment:
Can we get rid of the Gallery assessment set? I don't think anyone will want this in their course.